### PR TITLE
First step for formidable-core addon

### PR DESCRIPTION
**Pull request by xcambar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,14 +38,11 @@
...
     "ember-cli-sass": "5.3.1",
     "ember-cli-selectize": "0.5.5",
-    "ember-cli-showdown": "2.5.0",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^1.2.0",
-    "ember-cp-validations": "3.0.0-beta.0",
     "ember-data": "^2.4.2",
     "ember-drag-drop": "0.3.5",
     "ember-export-application-global": "^1.0.5",
-    "ember-feature-flags": "3.0.0",
```
Comment (GreatWizard) [inline package.json]: Can you remove featureFlags in the config/environnement.js?

**Comment by GreatWizard** [comment]
I think we need to move from devDeps to deps:

- emberx-file-input (use by field-file template)
- ember-radio-button (use by field-radio*)
- ember-truth-helpers (use by field-file template)

**Comment by GreatWizard** [comment]
Does the addon need a bower.json ? To add moment, moment-timezone, showndown...

**Comment by xcambar** [comment]
When we will extract it from the main component, it will need a bower.json. But it won't work if we do it now.

**Comment by xcambar** [comment]
No review?

**Comment by GreatWizard** [comment]
You right, I was so happy on friday I click a bit quicker =p

**Pull request merged**